open_samus_returns_rando/specific_patches/door_patches.py

```diff
@@ -98,8 +97,7 @@ def _patch_beam_bmsads(editor: PatcherEditor):
         editor.replace_asset(creature_bmsad_file, cr_bmsad)
 
     editor.add_new_asset(
-        "actors/props/doors/scripts/doors.lc", files_path().joinpath("doors.lua").read_bytes(),
-        lua_pkgs(editor.get_all_level_pkgs())
+        "actors/props/doors/scripts/doors.lc", files_path().joinpath("doors.lua").read_bytes(), []
```

Comment:
Same thing here as in the other PR: Moved lua files to the RomFS because they are read from there without issues and some didn't work in the pkg.